### PR TITLE
Docs: Fix kickstart syntax

### PR DESCRIPTION
**Pull request by kickoke**
#### Problem

`-token=TOKEN` is not the correct syntax for the kickstart script.
`-token TOKEN` is correct.

#### Solution
Removed `=` where it would mess up the kickstart script.

### CODE REVIEW

**Comment by Ferroin** [comment]
The particular cases here actually have nothing to do with the kickstart script itself, and any of the cases that involve `netdata-claim.sh` actually _need_ the `=` instead of a space.

I’m not sure about the `-W claim` stuff though, but I _think_ it also needs the `=` instead of spaces as well.

**Comment by ilyam8** [comment]
> I’m not sure about the -W claim stuff though, but I think it also needs the = instead of spaces as well.

From the [old helmchart release](https://github.com/netdata/helmchart/blob/e58a633dabde02bdf7e19978faa8940f4f53b61a/charts/netdata/templates/deployment.yaml#L64-L66)

```
              -W set2 cloud global enabled true
              -W set2 cloud global "cloud base url" "{{ .url }}"
              -W "claim -token={{ .token }} -rooms={{ .rooms }} -url={{ .url }}"
```

File: claim/README.md [hunk 1]
```diff
@@ -585,13 +585,13 @@ If a Netdata Agent is running, the Space's administrator can connect a node usin
...
 ```sh
--W "claim -token=TOKEN -rooms=ROOM1,ROOM2"
+-W "claim -token TOKEN -rooms ROOM1,ROOM2"
 ```
 
 For example:
 
 ```sh
-/usr/sbin/netdata -D -W "claim -token=MYTOKEN1234567 -rooms=room1,room2"
+/usr/sbin/netdata -D -W "claim -token MYTOKEN1234567 -rooms room1,room2"
```
Comment (ilyam8) [inline claim/README.md:594]: both were correct

Comment (DShreve2) [reply claim/README.md:594]: We should keep the original here, with the = ? 

@cpipilas , @ilyam8 , @Ferroin ?

Just double checking before I push this.

Comment (ilyam8) [reply claim/README.md:594]: Yes, we should keep the original.

Comment (ilyam8) [reply claim/README.md:594]: Yes.

Comment (DShreve2) [reply claim/README.md]: Thanks, @ilyam8 . Switched back! If you can put up an approval on this, I'll push this evening.

File: claim/README.md [hunk 2]
```diff
@@ -585,13 +585,13 @@ If a Netdata Agent is running, the Space's administrator can connect a node usin
 additional command line parameters:
 
 ```sh
--W "claim -token=TOKEN -rooms=ROOM1,ROOM2"
+-W "claim -token TOKEN -rooms=ROOM1,ROOM2"
```
Comment (ilyam8) [inline claim/README.md:588]:
```suggestion
-W "claim -token=TOKEN -rooms=ROOM1,ROOM2"
```

File: claim/README.md [hunk 3]
```diff
@@ -585,13 +585,13 @@ If a Netdata Agent is running, the Space's administrator can connect a node usin
...
 ```
 
 For example:
 
 ```sh
-/usr/sbin/netdata -D -W "claim -token=MYTOKEN1234567 -rooms=room1,room2"
+/usr/sbin/netdata -D -W "claim -token MYTOKEN1234567 -rooms=room1,room2"
```
Comment (ilyam8) [inline claim/README.md:594]:
```suggestion
/usr/sbin/netdata -D -W "claim -token=MYTOKEN1234567 -rooms=room1,room2"
```

**Review by ilyam8 — APPROVED** [review]

**Comment by ilyam8** [comment]
The OP is correct, but all the changes are netdata-claim.sh changes (not kickstart.sh). The netdata-claim.sh script syntax is `key=value`

https://github.com/netdata/netdata/blob/23fab077290b61507f65885b52307291f624ae9b/claim/netdata-claim.sh.in#L197-L203

This PR is wontfix.

**Pull request closed without merging**